app/assets/stylesheets/shared/footer.scss

```diff
@@ -54,23 +54,28 @@
   width: 100%;
 
   input {
-    flex-grow:1;
+    border-bottom-left-radius: $form-border-radius;
+    border-bottom-right-radius: 0;
+    border-top-left-radius: $form-border-radius;
+    border-top-right-radius: 0;
+
+    flex-grow: 1;
     height: 40px;
     vertical-align: top;
-    border-top-left-radius:$form-border-radius;
-    border-bottom-left-radius:$form-border-radius;
-    border-top-right-radius: 0px;
-    border-bottom-right-radius:0px;
   }
 
   button {
-    display:table-cell;
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: $base-border-radius;
+    border-top-left-radius: 0;
+    border-top-right-radius: $base-border-radius;
+
+    display: table-cell;
+    font-size: 1em;
+    line-height: 1.3;
```

Comment:
Properties should be ordered border-bottom-left-radius, border-bottom-right-radius, border-top-left-radius, border-top-right-radius, display, font-size, height, line-height, margin, padding, vertical-align